server/src/main/java/org/elasticsearch/action/admin/cluster/snapshots/restore/RestoreSnapshotRequest.java

```diff
@@ -61,6 +61,7 @@
     private boolean includeAliases = true;
     private Settings indexSettings = EMPTY_SETTINGS;
     private String[] ignoreIndexSettings = Strings.EMPTY_ARRAY;
+    private boolean skipOperatorOnly = false; // this field does not get serialised because it is always set locally by authz
```

Comment:
```suggestion
    private boolean skipOperatorOnlyState = false; // this field does not get serialised because it is always set locally by authz
```

I think adding a suffix is helpful because it makes it clear that the "only" refers to operator, not "skip".

By itself, `skipOperatorOnly` could mean "skip operator, but nothing else" which doesn't make sense if you know what "operator" means, but is still a cognitive hurdle.